pages/alerts.rst

```diff
@@ -284,11 +284,57 @@ Client URL
 .. image:: /images/alerts_pagerduty_notification.png
 
 
+.. _alert_notification_Slack Notification:
+
+Slack alert notification
+------------------------
+The Slack alert notification allows you to send notifications using your `Webhook URL <https://api.slack.com/messaging/webhooks>`__ to your slack workspace
+in response to the configured event definitions in your Graylog server.
+
+.. image:: /images/alerts_slack_notification.png
```

Comment:
Screen shot is very wide, which means the text comes out very small on the page.  Could we get a narrower screen shot?